packages/components/button/src/Button.test.tsx

```diff
@@ -0,0 +1,47 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { describe, expect, it, vi } from 'vitest'
+
+import { Button } from './Button'
+
+describe('Button', () => {
+  it('should render', () => {
+    // Given
+
+    // When
+    render(<Button>Hello World!</Button>)
+
+    // Then
+    expect(screen.getByText('Hello World!')).toBeInTheDocument()
```

Comment:
Use `getByRole('button', {name: ''})` instead 🙏🏻

Reply:
OMG. Maybe we should change the generator, cuz it's the 2nd time we got the same error